src/overlays/bankid/logo.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Can we use this SVG logotype for all Bank ID logos, and remove the binary images from the repo? SVG support is in every major browser nowadays.

Reply:
Yes, just a miss from my part.